targets/android-studio

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+REQUIRES='xiwi gtk-extra'
+DESCRIPTION='Installs Android Studio as an easily launchable xiwi application'
+HOSTBIN='startandroidstudio'
+
+### Append to prepare.sh:
+mkdir /etc/bash-completion.d #Must do this in order to avoid package management error while installing umake
+if [ $DISTRO != "ubuntu" ]; then
+  echo "deb http://ppa.launchpad.net/ubuntu-desktop/ubuntu-make/ubuntu yakkety main\n\
+     deb-src http://ppa.launchpad.net/ubuntu-desktop/ubuntu-make/ubuntu yakkety main" >> /etc/apt/sources.list
+  install ubuntu-make
+else
+  install ubuntu-make
```

Comment:
both branches end in `install ubuntu-make`, so this can be outside the if statement